lib/text/vtt_text_parser.js

```diff
@@ -955,10 +955,13 @@ shaka.text.VttTextParser = class {
       '&gt;': '>',
       '&quot;': '"',
       '&#39;': '\'',
+      '&nbsp;': ' ',
+      '&lrm;': '',
+      '&rlm;': '',
```

Comment:
Right-to-left mark in unicode is '\u{200f}'

This could be important for layout of multi-lingual text in some RTL languages.

Reply:
Done!